homeassistant/components/mysensors/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "MySensors",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
```

Comment:
The strings aren't updated. The English translation file is updated, but that's not the source of truth.